test/e2e/serial/tests/workload_placement.go

```diff
@@ -925,32 +925,33 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload placeme
 			podLabels := map[string]string{
 				"test": "test-rs",
 			}
-
-			rs := objects.NewTestReplicaSetWithPodSpec(replicaNumber, podLabels, map[string]string{}, fxt.Namespace.Name, rsName, corev1.PodSpec{
-				Containers: []corev1.Container{
-					{
-						Name:    "c0",
-						Image:   images.GetPauseImage(),
-						Command: []string{images.PauseCommand},
-						Resources: corev1.ResourceRequirements{
-							Limits: corev1.ResourceList{
-								corev1.ResourceCPU:    resource.MustParse("2"),
-								corev1.ResourceMemory: resource.MustParse("200Mi"),
-							},
+			rsContainers := []corev1.Container{
```

Comment:
OTOH this introduces coupling and require the reader to bounce up/down in the code to read the ReplicaSet definition.
So, while it is an improvement, is not obviously better on all fronts.

Reply:
note taken, thank you for highlighting this!